app/src/main/admin/actionTypes/ModellingGroupsTypes.ts

```diff
@@ -0,0 +1,36 @@
+import {ModellingGroup, ModellingGroupDetails, User} from "../../shared/models/Generated";
+
+export enum ModellingGroupTypes {
+    ADMIN_GROUPS_FETCHED = "ADMIN_GROUPS_FETCHED",
+    ADMIN_GROUP_DETAILS_FETCHED = "ADMIN_GROUP_DETAILS_FETCHED",
+    SET_CURRENT_ADMIN_GROUP = "SET_CURRENT_ADMIN_GROUP",
+    SET_CURRENT_ADMIN_GROUP_MEMBERS = "SET_CURRENT_ADMIN_GROUP_MEMBERS"
+}
+
+export interface AdminGroupsFetched {
+    type: ModellingGroupTypes.ADMIN_GROUPS_FETCHED;
```

Comment:
I'm sure you've just ported his terminology over from our code, but I have to say I'm not sure what 'admin group' means; any ideas @MartinEden ? Is it supposed to distinguish all groups from the current logged in user's groups? or is it informal namespacing again, to distinguish `admin` from `contrib`? I wonder if we can improve on these names...

Reply:
The latter - it's to distinguish the different group fetch capabilities between admin and contrib. May well be redundant now.

Reply:
Ok, in that case I'd suggest we drop the `Admin` prefix and see how we get on? We can revisit using ts namespaces if we run into confusion down the line?

Reply:
You are both absolutely right, it is kind of redundant and we can drop that Admin | Contrib namespacing in action naming. Were just initial naming, first what came to mind. Actions from different webapps parts should never intercross, as we only initialise one store per app part and trigger actions from only same correspondent app part. Only thing I am not sure now is how TypeScript will behave, but should be fine, will see.